.github/workflows/android.yml

```diff
@@ -0,0 +1,26 @@
+name: Android CI
```

Comment:
You have to make the name more meaningful like a _Verification pull request_ or something else

Reply:
Yes, I see but in the actions filter, we could see only `Android CI`
![image](https://user-images.githubusercontent.com/57491794/213219979-cc675c34-516a-47d2-8eaf-b6c8367a0d2a.png)
 instead of for example: 
![image](https://user-images.githubusercontent.com/57491794/213218811-49038970-2bc9-4871-b34d-2c1901b76cea.png)
We can have this if we have two flows one for the dev branch and one for the pull request. If it is not important for us you can leave it as is now :D

Reply:
Actually update the name is a good point, I will take care of it tomorrow